vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -1,165 +1,56 @@
 ---
 sidebarDepth: 2
 ---
-# Build and Publish a Project Bundle
+# Build and Publish a Bundle Project
 ## Overview
-This tutorial demonstrates how to deploy an existing Entando project directory into the [Local Hub](../../../docs/getting-started/concepts-overview.md#local-hub). This involves building a Docker image from your microservice, creating an Entando Bundle, checking your Bundle artifacts into Git, and deploying the Bundle into Kubernetes.
+This tutorial demonstrates how to deploy an existing Entando project directory into the [Local Hub](../../../docs/getting-started/concepts-overview.md#local-hub). The process involves: 
 
-The Entando CLI (ent) automates many of the tasks involved in deploying an Entando Bundle, but you can also choose to perform these tasks manually.
+1. Building a Docker image from a microservice
```

Comment:
The continuous present tense seems strange, like it's continuously happening. maybe change to simple present like build, define, push...

Reply:
agree. i changed to continuous because i was struggling to find a lead up i was happy with (don't start two sentences in a row with "this," don't add unnecessary or redundant verbiage, etc.)